swan-cern/files/swan_config_cern.py

```diff
@@ -39,34 +56,29 @@ def _init_eos_secret(self):
         except Exception as e:
             raise ValueError("Could not create required user credential")
 
-
         # ITHADOOP-819 - Ports need to be opened using service creation, and later assigning allocated service nodeport to a pod 
         # Create V1Secret with eos token
-        try:
-            secret_data = client.V1Secret()
-
-            secret_meta = client.V1ObjectMeta()
-            secret_meta.name = eos_secret_name
-            secret_meta.namespace = swan_container_namespace
-            secret_meta.labels = {
-                "swan_user": username
-            }
-            secret_data.metadata = secret_meta
-            secret_data.data = {}
-            secret_data.data['krb5cc'] = eos_token_base64
+        secret_data = V1Secret()
+
+        secret_meta = V1ObjectMeta()
+        secret_meta.name = eos_secret_name
+        secret_meta.namespace = swan_container_namespace
+        secret_meta.labels = {
+            "swan_user": username
```

Comment:
Could we align the label name to the same as the container?

Reply:
What name exactly?